metal/classifier.py

```diff
@@ -304,7 +307,7 @@ def _train_model(
                 optimizer.step()
 
                 # Keep running sum of losses
-                epoch_loss += loss.item()
+                epoch_loss += loss.item() * len(data[0])
```

Comment:
This feels obtuse; we average the loss, then multiply it back out, then average it again two turns later. I recognize that this change is just fixing a problem that was already there, but I think we should just work with the raw (sum) loss everywhere and then average once (for reporting/logging). Can you make a quick issue to remind us of this inconsistency?

Reply:
Actually, I'll go ahead and add the issue.

Reply:
Agree.